### ISSUE TITLE
using test set as validation set

### ISSUE THREAD

**Issue by bz36912** [keras-team/keras-io#1711]
### Issue Type

Bug

### Source

source

### Keras Version

Keras2.13

### Custom Code

Yes

### OS Platform and Distribution

Windows 11

### Python version

3.8.5

### Current Behavior?

In examples/vision/pointnet.py. You seem to use used your test set as validation set in line 263:
`model.fit(train_dataset, epochs=20, validation_data=test_dataset)`
This would cause the performance and predictive ability of your model to be overly optimistic, since the test set is not separate from your training process. The model is likely to perform poorly with unseen data/points.

### Standalone code to reproduce the issue or tutorial link

```shell
examples/vision/pointnet.py line 263:
`model.fit(train_dataset, epochs=20, validation_data=test_dataset)`
```

**Comment by simeetnayan81** [comment]
I'd like to contribute. @sachinprasadhs
